src/core/lib/iomgr/pollset_custom.cc

```diff
@@ -71,7 +71,7 @@ static grpc_error* pollset_work(grpc_pollset* pollset,
   grpc_millis now = grpc_core::ExecCtx::Get()->Now();
   size_t timeout = 0;
   if (deadline > now) {
-    timeout = deadline - now;
+    timeout = static_cast<size_t>(deadline - now);
```

Comment:
Code health question: why is timeout a size_t at all? Isn't that an awkward choice for representing a timeout? Why is that not also grpc_millis or something?

Reply:
It should be `grpc_millis` but pollset->poll() takes argument of type `size_t` (which is an awkward choice too..but cleaning all this up should probably be done in a different PR)..